src/components/Sections/SectionMarkdown.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/prop-types */
 import './SectionMarkdown.less';
 import '../../../assets/styles/railscasts.css';
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
There are tons of changes in this file, there is no good way to review this.
What was changed here?

Reply:
as i mentioned `modified MarkdownSection to react component to enable ref`, nothing else.